website/views.py

```diff
@@ -191,23 +192,27 @@ def get_dashboard_nodes(auth):
     """
     user = auth.user
 
-    contributed = user.node__contributed  # nodes user contributed to
-
-    nodes = contributed.find(
-        Q('category', 'eq', 'project') &
-        Q('is_deleted', 'eq', False) &
-        Q('is_registration', 'eq', False) &
-        Q('is_folder', 'eq', False)
+    nodes = Node.find_for_user(
+        user,
+        subquery=(
+            Q('category', 'eq', 'project') &
```

Comment:
Good point. Since this is such a small change, I think we can go ahead and make it in this PR. @erinspace Can you try out Brian's suggestion and make sure the Dashboard still has the correct nodes?